python/kfserving/kfserving/storage.py

```diff
@@ -192,7 +192,10 @@ def _download_blob(uri, out_dir: str):  # pylint: disable=too-many-locals
                     os.makedirs(dir_path)
 
             logging.info("Downloading: %s to %s", blob.name, dest_path)
-            block_blob_service.get_blob_to_path(container_name, blob.name, dest_path)
+            Path(os.path.dirname(dest_path)).mkdir(parents=True, exist_ok=True)
```

Comment:
With this do we still need the above code from 183 to 192 to create the directories?

Reply:
That's right, they were unnecessary, I removed those lines.